cg/server/dto/orders/orders_response.py

```diff
@@ -4,7 +4,7 @@
 
 
 class Order(BaseModel):
-    customer_id: int
+    customer_id: str
```

Comment:
I'd just call this field `customer_name`

Reply:
Reasonable, but might be confusing when we have a Customer.name field as well?

Reply:
Ah, we do?! Maybe we should display that instead in the frontend? Your call!

Reply:
Whenever guarantied to be unique I think `_id` should be used.

Reply:
I will keep it as the internal id for now - I think production more commonly refers to customers with custXXX than the name chosen by the customers themselves.